cinema/urls.py

```diff
@@ -0,0 +1,30 @@
+"""
+URL configuration for cinema project.
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/5.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+
+from django.contrib import admin
+from django.urls import path, include
+
+from cinema.views import movie_list, movie_detail
+
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("api/cinema/movies/", movie_list, name="cinema"),
+    path("api/cinema/movies/<int:pk>/", movie_detail, name="cinema_detail"),
```

Comment:
actually you had to create a separate app(name it cinema) and rename the main app. After that you should use api/cinema/ and include function to include urls from that cinema app